.github/workflows/e2e.yml

```diff
@@ -176,7 +171,7 @@ jobs:
     with:
       vlogger-image: ${{ needs.build.outputs.vlogger-image }}
       operator-image: ${{ needs.build.outputs.operator-image }}
-      vertica-image: ${{ needs.build.outputs.v2-vertica-image }}
+      vertica-image: ${{ needs.build.outputs.minimal-vertica-image }}
```

Comment:
Move of the v2 replacements have been minimal. Can you change a few to be the full image? It would be nice if roughly half of the vcluster tests used the minimal and half used the full image.